ansible_collections/arista/avd/plugins/action/anta_catalog_builder.py

```diff
@@ -0,0 +1,104 @@
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+from typing import Any, Dict, List, Tuple
```

Comment:
Please `from __future__ import annotations` and use the regular `dict, list, tuple` for typing.

Reply:
10/4